src/sage/calculus/transforms/dft.py

```diff
@@ -798,10 +798,10 @@ def idwt(self, other="haar", wavelet_k=2):
             sage: A = [RR(1) for i in J]
             sage: s = IndexedSequence(A,J)
             sage: t = s.dwt()
-            sage: t            # random arch dependent output
+            sage: t            # known bug, arch dependent output
```

Comment:
I don't think this is a "bug"

Reply:
See PR description.

Reply:
Yes, this is not a case that the PR description covers.